tests/manage/test_pvc_delete_verify_size_is_returned_to_backendpool.py

```diff
@@ -0,0 +1,179 @@
+size is returned to backend pool
+"""
+import logging
+import time
+import pytest
+
+from ocs import constants, defaults
+import ocs.exceptions as ex
+from tests import helpers
+from ocsci.testlib import tier1, ManageTest
+from utility import templating
+from resources import pod
+from ocs import ocp
+from tests.fixtures import (
+    create_rbd_storageclass, create_ceph_block_pool,
+    create_rbd_secret
+)
+
+logger = logging.getLogger(__name__)
+_templating = templating.Templating()
+
+PV = ocp.OCP(
+    kind='PersistentVolume', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+
+
+def check_ceph_used_space():
+    """
+    Check for the used space in cluster
+    """
+
+    # ToDo: after running IO's or deleting pod & pvc takes some time
+    time.sleep(20)
+    ct_pod = pod.get_ceph_tools_pod()
+    pods = ct_pod.exec_ceph_cmd(ceph_cmd="ceph status")
+    assert pods is not None
+    used = pods['pgmap']['bytes_used']
+    GB = (1024 * 1024 * 1024)
+    used_in_gb = used / GB
+    return used_in_gb
+
+
+def run_io(pod_obj):
+    """
+    Run io on the mount point
+    """
+
+    # Run IO's
+    pod_obj.exec_cmd_on_pod(
+        command="dd if=/dev/urandom of=/var/lib/www/html/dd_a bs=10M count=950"
+    )
+
+    # Verify data's are written to mount-point
+    mount_point = pod_obj.exec_cmd_on_pod(command="df -kh")
+    mount_point = mount_point.split()
+    used_percentage = mount_point[mount_point.index('/var/lib/www/html') - 1]
+    assert used_percentage > '90%'
+
+
+def verify_pv_not_exists(pv_name, cbp_name):
+    """
+    Ensure that pv does not exists
+    """
+
+    # validate on oc side
+    try:
+        assert not PV.get(pv_name)
+    except ex.CommandFailed as ecf:
+        assert "not found" in str(ecf)
+        logger.info(
+            f"Expected: pv should not be found "
+            f"after deleting corresponding pvc"
+        )
+
+    # Validate on ceph side
+    logger.info(f"Verifying pvc {pv_name} exists on backend")
+    _rc = False
+    ct_pod = pod.get_ceph_tools_pod()
+    pvc_list = ct_pod.exec_ceph_cmd(
+        ceph_cmd=f"rbd ls -p {cbp_name}", format='json'
+    )
+    for pvc in pvc_list:
+        if pvc == pv_name:
+            _rc = True
+            break
+    assert not _rc, (
+        f"pvc {pv_name} exists on backend"
+    )
+    logger.info(
+        f"Expected: pvc {pv_name} doesn't exist on backend after deleting pvc"
+    )
+
+
+def create_pvc_and_verify_pvc_exists(
+        sc_name, cbp_name,
+        desired_status=constants.STATUS_BOUND, wait=True
+):
+    """
+    Create pvc, verify pvc is bound in state and
+    pvc exists on ceph side
+    """
+
+    pvc_data = defaults.CSI_PVC_DICT.copy()
+    pvc_data['metadata']['name'] = helpers.create_unique_resource_name(
+        'test', 'pvc'
+    )
+    pvc_data['spec']['storageClassName'] = sc_name
+    pvc_data['spec']['resources']['requests']['storage'] = "10Gi"
+    global PVC_OBJ
+    PVC_OBJ = helpers.create_resource(**pvc_data, wait=False)
```

Comment:
you can pass wait here instead of waiting few lines below

Reply:
done